pkg/kubelet/apis/stats/v1alpha1/types.go

```diff
@@ -294,6 +298,13 @@ type FsStats struct {
 	InodesUsed *uint64 `json:"inodesUsed,omitempty"`
 }
 
+// DiskIoStats contains data about DiskIO
+type DiskIoStats struct {
+	// The time at which these stats were updated.
+	Time   metav1.Time                `json:"time"`
+	DiskIo *cadvisorapiv1.DiskIoStats `json:"diskio,omitempty"`
```

Comment:
We definitely don't want to include references to the cAdvisor api in the summary API